### DISCUSSION TITLE
How to login to public demo site https://sitecore-commerce-headstart-react-admin.vercel.app/ ?

### DISCUSSION THREAD

**Discussion by dannya** [Sitecore/Sitecore.Commerce.Headstart.ReactAdmin#210] [Q&A] [answered]
How to login to public demo site https://sitecore-commerce-headstart-react-admin.vercel.app/ ?

`Finally, log in as initialadminuser with the password Testingsetup123!` as mentioned in the project `README` didn't work for this site.

**Comment by crhistianramirez** [comment]
The instructions in the readme walk you through generating your own ordercloud marketplace (instance) that is separate from the marketplace used by that hosted site. We are not making access to that hosted site public at this time.

By following these instructions you will be able to log into your own ordercloud marketplace:
1. Run the seed command to generate your own marketplace (credentials are the same used when logging into https://ordercloud.io
2. Create a new `.env.local` file at the root of the project and populate the relevant settings from your newly created marketplace
    - `NEXT_PUBLIC_APP_NAME` - used as a prefix for cookie, value doesn't really matter
    - `NEXT_PUBLIC_OC_CLIENT_ID` - the clientID for your admin application
    - `NEXT_PUBLIC_OC_API_URL` - the ordercloud base url for your marketplace (varies by region/environment). Check your marketplace settings in ordercloud.io for this information
    - `NEXT_PUBLIC_OC_MARKETPLACE_ID` - the ID for your marketplace, found on ordercloud.io
    - `NEXT_PUBLIC_OC_MARKETPLACE_NAME` - the name for your marketplace, found on ordercloud.io
3. Run the app locally with `npm run dev`
4. Sign in with the default admin credentals  (user: initialadminuser, password: Testingsetup123!)

**Reply by dannya** [reply]
Hi @crhistianramirez , I am trying to login to your public demo site at the URL provided above.

**Reply by Elyseum** [reply]
Any update on this? There's little added value to hosting an instance online without being able to log in to it.

**Reply by robertsoniv** [reply, marked as answer]
In order to keep the marketplace data in working order for our own purposes, we disabled that login. I have brought it back, but with limited access, so the dashboard page will show a blank page. But you can at least view the buyer mgmt, order mgmt, and product mgmt now.
